### PR TITLE
python3Packages.ev3dev2: init at 2.1.0

### PR DESCRIPTION
**Pull request by emilytrau**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 130274` at 42137fe7 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.ev3dev2</li>
    <li>python39Packages.ev3dev2</li>
  </ul>
</details>

File: pkgs/development/python-modules/ev3dev2/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  src = fetchFromGitHub {
+    owner = "ev3dev";
+    repo = "ev3dev-lang-python";
+    rev = version;
+    sha256 = "XxsiQs3k5xKb+3RewARbvBbxaztdvdq3w5ZMgTq+kRc=";
+    fetchSubmodules = true;
+  };
+
+  prePatch = ''
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/ev3dev2/default.nix:19]:
```suggestion
  postPatch = ''
```

File: pkgs/development/python-modules/ev3dev2/default.nix [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  propagatedBuildInputs = [ pillow ];
+
+  checkPhase = ''
+    chmod -R g+rw ./tests/fake-sys/devices/**/*
+    python3 -W ignore::ResourceWarning tests/api_tests.py
+  '';
+
+  meta = with lib; {
+    description = "v2.x Python language bindings for ev3dev";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/ev3dev2/default.nix:31]:
```suggestion
    description = "Python language bindings for ev3dev";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/ev3dev2/default.nix [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+  postPatch = ''
+    echo "${version}\n" > RELEASE-VERSION
+  '';
+
+  propagatedBuildInputs = [ pillow ];
+
+  checkPhase = ''
+    chmod -R g+rw ./tests/fake-sys/devices/**/*
+    python3 -W ignore::ResourceWarning tests/api_tests.py
```
Comment (jonringer) [inline pkgs/development/python-modules/ev3dev2/default.nix:27]:
```suggestion
    ${python.interpreter} -W ignore::ResourceWarning tests/api_tests.py
```
to support alt interpreters

**Pull request merged**